docs/results.md

```diff
@@ -29,6 +29,22 @@ Version   | Approach    | Micro f1 | Description
 2020.06.1 | ensemble of tfidf and scibert | 0.75 | same as 2020.06.0 with threshold 0.3
 2020.06.2 | tfidf-adaboost | 0.63 | adaboost after tfidf
 2020.06.3 | tfidf-gboost | 0.66 | same as 2020.06.2 but with gradient boosting instead
+2020.10.0 | tfidf-svm | 0.72 | same as 2020.05.2 with train data including lay summary
+2020.10.1 | tfidf-svm | 0.72 | same as 2020.05.2 with train data including lay summary and research question
+2020.10.2 | tfidf-svm | 0.69 | same as 2020.05.2 with train data including lay summary but testing on title+synopsis
+2020.10.3 | tfidf-svm | 0.69 | same as 2020.05.2 with train data including lay summary and research question but testing on title+synopsis
```

Comment:
doesnt seem to inc research q in the config `text_cols=Title,Synopsis,Lay Summary`

Reply:
139adcc also tested the results which stayed the same. this is probably because these files are not recreated if they exist so it was probably created from another config correctly